src/doc/rustc/src/lints/index.md

```diff
@@ -39,19 +39,32 @@ provides detailed information and an opportunity for feedback. This gives
 users some time to fix the code to accommodate the change. After some time,
 the warning may become an error.
 
-The following is an example of what a future-incompatible looks like:
-
-```text
-warning: borrow of packed field is unsafe and requires unsafe function or block (error E0133)
-  --> lint_example.rs:11:13
-   |
-11 |     let y = &x.data.0;
-   |             ^^^^^^^^^
-   |
-   = note: `#[warn(safe_packed_borrows)]` on by default
-   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
-   = note: for more information, see issue #46043 <https://github.com/rust-lang/rust/issues/46043>
-   = note: fields of packed structs might be misaligned: dereferencing a misaligned pointer or even just creating a misaligned reference is undefined behavior
+The following is an example of what future-incompatible code looks like:
+
+```bash
+$ cat main.rs
+#[repr(packed)]
+struct Packed {
+    f1: u8,
+    f2: u16,
+}
+
+fn main() {
+    let packed = Packed { f1: 1, f2: 2 };
+    let _ = &packed.f2; // This line triggers the warning
+}
+
+$ rustc main.rs
+
+error[E0793]: reference to packed field is unaligned
```

Comment:
that's not actually a future incompat lint now, it's just an error, you need a different example. search for `@future_incompat` in the compiler source to find examples

Reply:
Thanks @Nilstrieb , I learned a lot hunting these down.